### ISSUE TITLE
Update Font Awesome to version 5.14 or newer

### ISSUE THREAD

**Issue by wpniall** [woocommerce/storefront#1751]
## Describe the solution you'd like
Per the [Could we get fontawesome updated to 5.14](https://wordpress.org/support/topic/could-we-get-fontawesome-updated-to-5-14/) request on the WordPress.org forums, it would be great to have the Font Awesome version for Storefront updated to version 5.14 or newer to add support for TikTok icons and more.

## Additional context
It looks like [Font Awesome added the Tik Tok icon in version 5.13.1](https://github.com/FortAwesome/Font-Awesome/issues/14340), then made [breaking changes to the icon and a few other icons in version 5.14](https://fontawesome.com/v5/changelog/latest), so it seems a better idea to update Storefront's Font Awesome version to 5.14 or higher.

**Comment by GlennMartin1** [comment]
As of 8/6/2021, the current version is 5.15.4. See https://fontawesome.com/v5/changelog/latest

**Comment by wpniall** [comment]
Yes, you're right @GlennMartin1! I specified version 5.14 or higher in case there's a valid reason not to use the latest 5.15.4 version, as 5.14 is the earliest version containing the requested icon updates.

**Comment by JethroHazelhurst** [comment]
Hi there! Just wanted to bump this as this update to the Fontawesome version has been requested for a long time now. We depend on TikTok for most of our engagement (beauty/skincare product segment) so to not be able to have the TikTok icon on our website is a real handicap. This has been requested before [here](https://wordpress.org/support/topic/could-we-get-fontawesome-updated-to-5-14/) and [here](https://wordpress.org/support/topic/storefront-fontawesome-tiktok-icon-when/). Please, please, please can we get this updated in a future release? Our main social channel is essentially disconnected until we are able to show this major platform icon.

**Comment by theabhig** [comment]
Just adding a request for the Tiktok icon specifically - 7690890-zen
